test/helper/passwordHelpers.test.js

```diff
@@ -0,0 +1,38 @@
+const chai = require('chai');
+const { expect } = chai;
+const { checkPasswordStrength, passwordsMatch } = require('../../src/helper/passwordHelpers');
+
+describe('passwordHelpers tests', function () {
+	describe('checking validity of password strength', function () {
+		it('should return false if the passwords do NOT match the requirements', function () {
+			const someWeekPasswords = ['somepassword', 'somepassword1', 'somepassword1!', 'somepassword1!@#', 'afsf@#$'];
```

Comment:
Codacy found an issue: [This line has a length of 121. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5748939)